src/com/xmslyz/luhn/main/OptionMenu.java

```diff
@@ -0,0 +1,43 @@
+package com.xmslyz.luhn.main;
+
+import com.xmslyz.luhn.logic.LuhnAlgorithmEngine;
+import java.util.Scanner;
+
+public class OptionMenu extends LuhnAlgorithmEngine {
```

Comment:
Niepotrzebne dziedziczenie:
```suggestion
import java.util.Scanner;

public class OptionMenu {
```